markdown/spring-cloud-stream-beyond/spring-cloud-stream-beyond.md

```diff
@@ -672,6 +678,143 @@ You'll note that you received 1 message on the default binding destination of `m
 
 ![Batch Publish 2](img/batchPublishTryMe2.webp)
 
+## Batch Consumption
+Duration: 0:07:00
+
+Batch consumers are a feature that allows consuming messages in batches rather than individually. This feature is particularly useful when dealing with high-volume data processing scenarios where processing messages individually may introduce performance overhead.
+
+Batching of messages is applied only when the `spring.cloud.stream.bindings.<binding-name>.consumer.batch-mode` is set to true. The number of messages in a batch is dependent on the availability of messages in the queue and the timeout to receive messages from the queue.
```

Comment:
Looking at the preview this doesn't show up properly, it shows up like "spring.cloud.stream.bindings..consumer.batch-mode" ....notice the ".." in there and no `<binding-name>`. Please resolve

Reply:
Usual < and > chaarcter issue, fixed it.